### ISSUE TITLE
Gyro_sync causes motors to spin on power on

### ISSUE THREAD

**Issue by DzikuVx** [iNavFlight/inav#260]
I've seen something scary today. With gyro_sync=ON motors started to spin when power connected. Good thing they were spinning on low power... Configuration
- SPRacingF3
- OneShot125
- GPS
- OLED display

Disabling gyro_sync fixed the issue. BTW, I'm using gyro_sync=ON and OneShot125 on other quad, but without GPS, Mag and OLED display

**Comment by theArchLadder** [comment]
Strange and scary! I'm using the same setup except the oled, and that has never happened for me. You use aux arming, right? I use stick arming...

Did it happen on every startup or just once?

**Comment by DzikuVx** [comment]
On every startup. And yes, aux arming.

**Comment by DzikuVx** [comment]
And BTW it was gyro_sync_denom=1

**Comment by theArchLadder** [comment]
I also use

```
gyro_sync=ON
gyro_sync_denom=1
```

Perhaps you could try and see if you still have this problem with stick arming, or without oled?
I have used aux arming on some occasions, but never had the motors spin up when i connected the battery...

**Comment by digitalentity** [comment]
Weird. Can you please test if it's related to oled or some other feature?
I'm flying oneshot with gyro sync at 1khz - no issues.

**Comment by DzikuVx** [comment]
I will try to confirm the cause.

**Comment by DzikuVx** [comment]
So far, as I do not have access to any F3 board, I was unable to confirm that on Flip32 with display, GPS, gyro_sync=ON and denom=2 since firmware prohibits looptime faster than 2000us

**Comment by DzikuVx** [comment]
Although I'm on vacation, I took my 250 and since it's raining I've decided to update few things and unfortunately I had the same issue one more time. This time on completely different setup:
- SPRacingF3 (Chinese)
- Emax Nano 12A ESC (They say it's BLHeli by EMAX)
- OneShot125
- No GPS, Display

After setting looptime 1000

```
set gyro_sync = ON
set gyro_sync_denom = 8
set gyro_lpf = 256HZ
```

Motors started to spin on boot. Setting `set gyro_sync_denom = 12` (looptime 1500) problem went away. 

In both cases it was EMAX ESC with stock firmware. So I tend to think it is something with ESCs by EMAX.
